internal/gengapic/generator.go

```diff
@@ -316,12 +316,13 @@ func (g *generator) autoPopulatedFields(servName string, m *descriptor.MethodDes
 	var apfs []string
 	// Find the service config's AutoPopulatedFields entry by method name.
 	for _, s := range g.serviceConfig.GetPublishing().GetMethodSettings() {
-		if s.GetSelector() == g.fqn(m) {
+		mfqn := g.fqn(m)
```

Comment:
I should've been more clear in my comment. Moving the `g.fqn(m)` call outside of the loop b.c it doesn't change inside of the loop, so calling it there repeatedly is a waste of cycles.

Reply:
Actually I understood perfectly the first time, but I simply put the new variable on the wrong line! Ugh!